### PR TITLE
Rebaseline performance tests

### PR DESCRIPTION
**Pull request by hegyibalint**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -25,4 +25,4 @@ systemProp.gradle.internal.testdistribution.writeTraceFile=true
 systemProp.gradle.internal.testdistribution.queryResponseTimeout=PT20S
 
 # Default performance baseline
-defaultPerformanceBaselines=8.1-commit-f6b887c2
+defaultPerformanceBaselines=85ebea1
```
Comment (wolfs) [inline gradle.properties:28]: That doesn't look right. It should be something like `8.1-commit-...` or `8.2-commit-...`.

Moreover, the PR should always reference a performance issue why we are rebasing.

**Review by wolfs — CHANGES_REQUESTED** [review]

File: gradle.properties (hunk 1 above)
Comment (hegyibalint) [reply gradle.properties:28]: - Docs are not correct in this case, I tried to follow it to the letter.
- Issue referenced from private

Comment (wolfs) [reply gradle.properties:28]: We have some docs what the baseline can look like: https://bt-internal-docs.grdev.net/gbt/performance/how-performance-test-works/#baseline

Though the step-by-step instructions don't clarify what you need to do with the short commit id you obtain.

Comment (hegyibalint) [reply gradle.properties:28]: Found it meanwhile, and I don't understand it.

> A commit version, e.g. 8.0-commit-6d0169890d7.

What is a commit version? :D

Comment (hegyibalint) [reply gradle.properties:28]: This time, I haven't found another level of something, this needs to be elaborated.

Comment (wolfs) [reply gradle.properties:28]: A commit version is a version of Gradle pointing to a specific commit. The format is `<base-version>-commit-<short commit hash>`.

Comment (hegyibalint) [reply gradle.properties:28]: But where `base-version` comes from? It doesn't look like something coming from [releases.json](https://github.com/gradle/dotorg-site/blob/71757280cbc7f6bf40eed83c36a1d315f15e4786/_data/releases.json) nor [releases.xml](https://github.com/gradle/web/blob/34a35380aca7f9c51d1cfdfc2b2e4207932e0868/data/releases.xml).

File: gradle.properties [hunk 2]
```diff
@@ -25,4 +25,4 @@ systemProp.gradle.internal.testdistribution.writeTraceFile=true
 systemProp.gradle.internal.testdistribution.queryResponseTimeout=PT20S
 
 # Default performance baseline
-defaultPerformanceBaselines=8.1-commit-f6b887c2
+defaultPerformanceBaselines=8.2-commit-85ebea1
```
Comment (hegyibalint) [inline gradle.properties:28]: @wolfs let's iterate. I guess it's the [version.txt](https://github.com/gradle/gradle/blob/master/version.txt) where this comes from, plus a short-SHA. This should be a rebaseline on the top commit.

Should this even work?

Comment (wolfs) [reply gradle.properties:28]: Yes, the base version comes from the `version.txt` the commit you want to rebaseline on is pointing to.

Our build then builds the commit distribution by checking out the commit and assigning the corresponding version to it.

Comment (wolfs) [reply gradle.properties:28]: If the version the commit is pointing to is `8.2`, then yes.

Comment (hegyibalint) [reply gradle.properties:28]: It should

https://github.com/gradle/gradle/blob/85ebea10e4e150ce485184adba811ed3eeaa2622/version.txt#L1

Then the question is if this _makes sense_? 
It makes for me, as what we are trying to achieve here is to make master green by pushing the performance issues aside (to be fixed later).

Comment (wolfs) [reply gradle.properties:28]: I think it makes sense. One question from me is whether the performance regression shows up on `release` as well, and if it does then we should rebaseline there to a commit on `release`.

Comment (hegyibalint) [reply gradle.properties:28]: Release seems to be fine: https://builds.gradle.org/buildConfiguration/Gradle_Release_Check_PerformanceTestTestLinux_Trigger?branch=&mode=builds#all-projects

**Review by wolfs — APPROVED** [review]

**Comment by hegyibalint** [comment]
@wolfs thanks for the help!

**Comment by hegyibalint** [comment]
@bot-gradle test and merge please

**Pull request merged**